README.md

```diff
@@ -64,27 +98,31 @@ The remaining infrastructure is managed internally with TerraForm.
    cd ..
    ```  
 
-4) Edit `deployments/.active/environment` and
+3) Edit `deployments/.active/environment` and
    `deployments/.active/environment.local` according to the comments in there.
 
-5) Load the environment:
+___
 
-   ```
-   source environment
-   ```
-   
-   Examine the output.
+### 1.4 PyCharm configuration specifics.
+Configuring the working environment in PyCharm can improve the IDEs assistance. This allows for the whole environment to be synchronize and eliminate issues pertaining to dependencies.   
 
-6) Install the development prerequisites
-   
-   ```
-   pip install -r requirements.dev.txt
-   ```
+- Connect python hook for `environment` in PyCharm
+    ```bash
+    python scripts/envhook.py install
+    ```
 
-7) Run `make`. It should say `Looking good!` If one of the sanity checks fails,
-   address the complaint and repeat. The various sanity checks are defined in
-   `common.mk`.
+- Under `Settings -> Project interpreter` select the virtual environment created above.
```

Comment:
`Project` and `Interpreter` should be italic and there should be an emdash (wide dash) between them. Alt-Shift-Dash. Should look like this: —